src/main/java/org/cascadebot/cascadebot/commands/management/FlagsCommand.java

```diff
@@ -1,26 +1,16 @@
 package org.cascadebot.cascadebot.commands.management;
 
-import net.dv8tion.jda.api.EmbedBuilder;
-import net.dv8tion.jda.api.entities.Guild;
 import net.dv8tion.jda.api.entities.Member;
-import org.apache.commons.lang3.StringUtils;
-import org.cascadebot.cascadebot.UnicodeConstants;
 import org.cascadebot.cascadebot.commandmeta.CommandContext;
 import org.cascadebot.cascadebot.commandmeta.ICommandMain;
 import org.cascadebot.cascadebot.commandmeta.Module;
-import org.cascadebot.cascadebot.data.objects.Flag;
-import org.cascadebot.cascadebot.data.objects.GuildData;
-import org.cascadebot.cascadebot.messaging.MessagingObjects;
 import org.cascadebot.cascadebot.permissions.CascadePermission;
-import org.cascadebot.cascadebot.utils.FormatUtils;
-
-import java.util.Arrays;
-import java.util.stream.Collectors;
 
 public class FlagsCommand implements ICommandMain {
```

Comment:
Maybe the flags command can become `;tier`? It would just print the result of Tier#getTierString

Reply:
Commands need a re-think so me and binary have decided for this pr to not worry about commands